app/cdap/components/GridTable/types.ts

```diff
@@ -48,3 +48,21 @@ export interface IHeaderNamesList {
 export interface IType {
   [key: string]: string | number;
 }
+
+interface IGeneralObjectRecord {
+  [key: string]: number | string;
+}
+
+interface IGeneral {
+  general?: IGeneralObjectRecord;
+  types?: IGeneralObjectRecord;
+}
+
+export interface IStatistics {
```

Comment:
I'm not sure I see too much value in these types - especially with them named as they are. An object type with two optional attributes doesn't provide much value - a general object type also doesn't provide much value for type checking.

Reply:
Done